crowbar_framework/app/models/role_object.rb

```diff
@@ -206,7 +206,8 @@ def self.find_roles_by_search(search)
 
   def self.find_role_by_name(name)
     begin
-      return RoleObject.new Chef::Role.load(name)
+      # get role directly from CouchDB, skip Chef API
+      return RoleObject.new Chef::Role.cdb_load(name)
```

Comment:
as we are hitting the database, we wont get those errors anymore no?

Reply:
It's REST interface so I guess we should get the same kind of errors.

Reply:
+1